pkg/api/v1/resource/helpers.go

```diff
@@ -26,8 +26,14 @@ import (
 	"k8s.io/apimachinery/pkg/api/resource"
 )
 
-// GetResourceRequestQuantity finds and returns the request quantity for a specific resource.
-func GetResourceRequestQuantity(pod *v1.Pod, resourceName v1.ResourceName) resource.Quantity {
+type resourceType string
+
+const (
+	resourceTypeReuqest resourceType = "reuqest"
```

Comment:
request* in both the variable name and value

Reply:
Done!